bedrock/externalpages/templates/externalpages/pocket/welcome.html

```diff
@@ -0,0 +1,17 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/platform-pages.html" %}
+
+{% set platform_image = 'img/externalpages/pocket/welcome-bookmarklet.jpg' %}
+
+{% block page_title %} Pocket for Your Browser {% endblock %}
+
+{% block page_desc %} When browsing the web, simply push this button in your Bookmarks Bar to add pages to your Pocket list. {% endblock %}
+
+{% block page_cta %} Log in {% endblock %}
```

Comment:
The [original page](https://getpocket.com/en/welcome/) didn't have styling for the CTA, but since this page looked similar to the rest of the platform pages, I just made it have the same CTA design.